conviva/src/main/java/com/bitmovin/analytics/conviva/ContentMetadataBuilder.java

```diff
@@ -43,42 +44,35 @@ public void setPlaybackStarted(boolean value) {
         playbackStarted = value;
     }
 
-    public ContentMetadata build() {
+    public Map<String, Object> build() {
         if (!playbackStarted) {
             // Asset name is only allowed to be set once
-            if (contentMetadata.assetName == null) {
-                contentMetadata.assetName = getAssetName();
+            if (!contentMetadata.containsKey(ConvivaSdkConstants.ASSET_NAME)) {
+                contentMetadata.put(ConvivaSdkConstants.ASSET_NAME, getAssetName());
             }
 
-            contentMetadata.viewerId = getViewerId();
-            contentMetadata.streamType = ObjectUtils.defaultIfNull(
-                    metadataOverrides.getStreamType(),
-                    metadata.getStreamType());
-
-            contentMetadata.applicationName = ObjectUtils.defaultIfNull(
-                    metadataOverrides.getApplicationName(),
-                    metadata.getApplicationName());
```

Comment:
~What happened to `StreamType` and `ApplicationName`? Are they no longer needed? 🤔~

EDIT: found them 🙂 👍